### PR TITLE
[WIP] [KO-korean] README translation (Chapter 1, 5)

### PR DESCRIPTION
**Pull request by SoniaComp**
[ My Three Translation Principles ]
1. I translated the literal translation in principle.
2. Change to a term widely used in the industry.
3. I paraphrased only when its meaning is ambiguos.

[#197] Chapter 1, 5
- [x] Introduction base README
  - [x]  Defining Data Science README
  - [x] Defining Data Science assignment
  - [x] Ethics README
  - [x]  Ethics assignment
  - [x] Defining Data README
  - [x]  Defining Data assignment
  - [x] Stats and Probability README
  - [x]  Stats and Probability assignment
- Working with Data base README (assigned by #167)
- [x] Vizualization base README
- Data Science Life cycle
- [x] Data Science in the Cloud base README
  - [x] Intro README
  - [x] Intro assignment
  - [x] Low Code README
  - [x] Low Code assignment
  - [x] Azure README
  - [x] Azure assignment
- Data Science in the Wild base

### CODE REVIEW

**Comment by jlooper** [comment]
Wonderful! Could you turn this into a draft? 🙏

**Comment by SoniaComp** [comment]
Sure! Thank you for your comment 😊

**Comment by SoniaComp** [comment]
Sorry for late upload. I think I will finish it until this Sunday.

**Comment by SoniaComp** [comment]
I translated all assignments, and READMEs of Chapter 1 and 5.
Sorry for late upload and thank you for giving me chance to contribute to this project.

File: 1-Introduction/translations/README.ko.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# 데이터 과학(Data Science) 소개
+
+![활용중인 데이터](../images/data.jpg)
+> 촬영 작가: <a href="https://unsplash.com/@dawson2406?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Stephen Dawson</a> on <a href="https://unsplash.com/s/photos/data?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+
+이 수업에서는 데이터 과학이 어떻게 정의되는지 알아보고 데이터 과학자가 고려해야 하는 윤리적인 사항들에 대해 배웁니다. 또한 데이터가 어떻게 정의되는지 배우고 데이터 과학의 핵심 학문 영역인 확률과 통계에 대해서 간단히 배우게 됩니다.
+
+### 주제
+
+1. [데이터 과학 정의](../01-defining-data-science/README.md)
```
Comment (paladique) [inline 1-Introduction/translations/README.ko.md:10]: These can point to the translated ones now :)

**Review by paladique — CHANGES_REQUESTED** [review]
Wow, thanks so much for your work on this!! Just a few small comments on this and it will be ready to merge once resolved.

File: 5-Data-Science-In-Cloud/translations/README.ko.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+> [Unsplash](https://unsplash.com/s/photos/cloud?orientation=landscape)의 [Jelleke Vanooteghem](https://unsplash.com/@ilumire)의 사진
+
+빅 데이터로 데이터 과학을 수행할 때 클라우드를 사용하면 판도를 뒤집어 놓을 수 있습니다(game changer). 다음 세 강의에서는 클라우드가 무엇이며 왜 매우 유용할 수 있는지 알아보겠습니다. 우리는 또한 심부전 데이터셋을 탐색하고 누군가가 심부전에 걸릴 확률을 평가하는 데 도움이 되는 모델을 구축할 것입니다. 클라우드의 힘을 사용하여 두 가지 방식으로 모델을 훈련, 배포 및 사용합니다. 첫번째 방법은 로우 코드(Low code)/노 코드(No code) 방식으로 사용자 인터페이스만 사용하는 방법, 두번째 방법으로는 Azure 기계 학습 소프트웨어 개발자 키트(Azure ML SDK)를 사용하는 방법이 있습니다.
+
+![프로젝트 스키마](../19-Azure/images/project-schema.PNG)
+
+### 주제
+
+1. [데이터 사이언스에 클라우드를 사용하는 이유](17-Introduction/README.md)
```
[the remark this answers is not in the document]
Comment (SoniaComp) [reply 5-Data-Science-In-Cloud/translations/README.ko.md]: Thank you for your comment!  😊 I all changed codes related with this issue.

File: 1-Introduction/translations/README.ko.md (hunk 1 above)
Comment (SoniaComp) [reply 1-Introduction/translations/README.ko.md]: Either this issue was solved! Thank you so much!!

**Comment by SoniaComp** [comment]
@paladique Thanks a lot for your checking 🙏 I solved all of the problems, and you can resolve conversation 😊

**Review by paladique — APPROVED** [review]

**Pull request merged**